### DISCUSSION TITLE
天翼云盘似乎禁止了我的账号使用alist挂载

### DISCUSSION THREAD

**Discussion by victorious210** [AlistGo/alist#7671] [General]
两种驱动我都按照官方文档和建议尝试了多次，始终提示需要验证码或者验证码错误。
如果尝试次数或者错误次数过多 ，甚至会直接报废当前账号的密码或者禁止使用账号密码登录。
我换了一个账号后，立刻恢复了正常，不知道有没有朋友遇到类似的情况？

**Comment by Wallemuse** [comment]
你去天翼里面把登录需要短信验证关掉是试试啊

**Reply by victorious210** [reply]
和这关系不大，我另外一个账号也是开的，但是能正常挂载，感觉就是被官方标记了

**Comment by shanhai2333** [comment]
有没有可能就是频繁失败风控了

**Comment by victorious210** [comment]
已经解决，原因竟然是电脑网页端无法正常修改密码，换到手机端即可。
